numba/cuda/models.py

```diff
@@ -23,4 +24,18 @@ def __init__(self, dmm, fe_type):
         super().__init__(dmm, fe_type, be_type)
 
 
+@register_default(types.Float)
```

Comment:
I think it will be doing that. In the long term I think we will need to allow targets to have their own `DataModelManager` to avoid this. Perhaps this ought to be brought up in the dev meeting?

Reply:
I think it'd be good to raise that at the next meeting, thanks.